requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 --index-url https://pypi.devinfra.sentry.io/simple
 
+covdefaults>=2.3.0
```

Comment:
the plugin needs to also be enabled to work

Reply:
Heh. In all the confusion over the getsentry pypi I lost track of that, you're right. 🤦🏻‍♀️